specification/attestation/resource-manager/Microsoft.Attestation/stable/2020-10-01/attestation.json

```diff
@@ -591,29 +599,224 @@
       "properties": {
         "location": {
           "type": "string",
-          "description": "The supported Azure location where the attestation service instance should be created."
+          "description": "The supported Azure location where the attestation provider should be created."
         },
         "tags": {
           "type": "object",
           "additionalProperties": {
             "type": "string"
           },
-          "description": "The tags that will be assigned to the attestation service instance."
+          "description": "The tags that will be assigned to the attestation provider."
         },
         "properties": {
           "$ref": "#/definitions/AttestationServiceCreationSpecificParams",
-          "description": "Properties of the attestation service instance"
+          "description": "Properties of the attestation provider"
         }
       }
     },
     "AttestationServiceCreationSpecificParams": {
-      "description": "Client supplied parameters used to create a new attestation service instance.",
+      "description": "Client supplied parameters used to create a new attestation provider.",
       "properties": {
         "policySigningCertificates": {
           "description": "JSON Web Key Set defining a set of X.509 Certificates that will represent the parent certificate for the signing certificate used for policy operations",
           "$ref": "../../../../../common-types/rfcs/rfc7517.json#/definitions/JSONWebKeySet"
         }
       }
+    },
+    "PrivateEndpointConnection": {
```

Comment:
Can we reuse common definition in "../../../../../common-types/resource-management/v1/privatelinks.json"?
If your service implementation is match with common definition

Reply:
7 lint warnings? I see the Java SDK error but that's the only one I'm seeing.

Reply:
⚠️ R2001 - AvoidNestedProperties | Consider using x-ms-client-flatten to provide a better end user experienceNew: Microsoft.Attestation/stable/2020-10-01/attestation.json#L678
-- | --
⚠️ R4021 - DescriptionAndTitleMissing | 'PrivateLinkConnectionItemProperties' model/property lacks 'description' and 'title' property. Consider adding a 'description'/'title' element. Accurate description/title is essential for maintaining reference documentation.New: Microsoft.Attestation/stable/2020-10-01/attestation.json#L643
⚠️ R4021 - DescriptionAndTitleMissing | 'provisioningState' model/property lacks 'description' and 'title' property. Consider adding a 'description'/'title' element. Accurate description/title is essential for maintaining reference documentation.New: Microsoft.Attestation/stable/2020-10-01/attestation.json#L646
⚠️ R4021 - DescriptionAndTitleMissing | 'privateEndpoint' model/property lacks 'description' and 'title' property. Consider adding a 'description'/'title' element. Accurate description/title is essential for maintaining reference documentation.New: Microsoft.Attestation/stable/2020-10-01/attestation.json#L649
⚠️ R4021 - DescriptionAndTitleMissing | 'privateLinkServiceConnectionState' model/property lacks 'description' and 'title' property. Consider adding a 'description'/'title' element. Accurate description/title is essential for maintaining reference documentation.New: Microsoft.Attestation/stable/2020-10-01/attestation.json#L658
⚠️ R4021 - DescriptionAndTitleMissing | 'properties' model/property lacks 'description' and 'title' property. Consider adding a 'description'/'title' element. Accurate description/title is essential for maintaining reference documentation.New: Microsoft.Attestation/stable/2020-10-01/attestation.json#L678